Tests/Smoketest3.xctestplan

```diff
@@ -38,7 +38,6 @@
         "HistoryTests\/testClearHistoryFromSettings()",
         "HistoryTests\/testClearPrivateDataButtonDisabled()",
         "HistoryTests\/testClearRecentlyClosedHistory()",
-        "HistoryTests\/testDeleteHistoryEntryBySwiping()",
```

Comment:
Ok for rebalancing the test suites. Smoketest3 has been finishing earlier than other smoke tests.